client/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@g-loot/react-tournament-brackets": "^1.0.30",
```

Comment:
Seems to be [deprecated](https://github.com/g-loot/react-tournament-brackets/issues/78).